Validation/RecoEgamma/plugins/ElectronMcSignalValidator.cc

```diff
@@ -3556,6 +3586,7 @@ void ElectronMcSignalValidator::analyze(const edm::Event &iEvent, const edm::Eve
     h1_mc_Phi_matched->Fill(mcIter->phi());
     h1_mc_AbsEta_matched->Fill(std::abs(mcIter->eta()));
     h1_mc_Eta_matched->Fill(mcIter->eta());
+    h1_mc_Eta_Extended_matched->Fill(mcIter->eta());
```

Comment:
You are duplicating the previous histo, only with a slightly wider x-range: is it really what you want?
One single histo with the widest range cannot suit as well?

Reply:
Hi Andrea,
yes, it is intended. The idea is to introduce a new category of plots to monitor electrons in 2.5<|eta|<3 without affecting the rest of the plots. Florian

Reply:
> Hi Andrea, yes, it is intended. The idea is to introduce a new category of plots to monitor electrons in 2.5<|eta|<3 without affecting the rest of the plots. Florian

Well, the eta distributions are exactly the same in the two pair of plots that I pointed out, with only the x-axis range of the two histos being extended. I will not insist, of course: they are just two fully redundant validation histos which are added over a much larger number. Still, it doesn't seem really justified to me.